site/app/config/parameters.yml.dist

```diff
@@ -7,6 +7,8 @@ parameters:
     database_user:     librecores
     database_password: ~
 
+    mailer_auth_mode: login #the type of authentication the mailer is checking eg: 'login'
```

Comment:
I'm not sure if this is required, the plan is to go with AWS SES, which supports plain auth`.